source/2019-07-05-the-ember-times-issue-105.md

```diff
@@ -14,6 +14,7 @@ help design the **Ember skill tree** 🌱,
 RFC to **support populating head tag** ⛑️,
 RFC to add `load` hook to Route 🎣
```

Comment:
Not related to your PR, Chris, but can we add commas to end of the lines 15 & 16 to separate items? Thanks.

Reply:
added!